### PR TITLE
fix python runner oauth2

### PR DESCRIPTION
**Pull request by freeznet**
Fixes https://github.com/streamnative/function-mesh/issues/489

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation

Make changes according to https://github.com/streamnative/function-mesh/issues/489#issuecomment-1300713980

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

Check the box below.

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [x] `no-need-doc` 
  
  (Please explain why)
  
- [ ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

File: .ci/tests/integration-oauth2/cases/py-download-function-legacy/verify.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```
Comment (jiangpengcheng) [inline .ci/tests/integration-oauth2/cases/py-download-function-legacy/verify.sh:1]: this need to be added in https://github.com/streamnative/function-mesh/blob/master/.ci/tests/integration/e2e.yaml#L141

Comment (freeznet) [reply .ci/tests/integration-oauth2/cases/py-download-function-legacy/verify.sh:1]: fixed

File: .ci/tests/integration-oauth2/cases/py-download-function-legacy/manifests.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      - name: secret-broker-0
+        secret:
+          defaultMode: 420
+          secretName: sn-platform-oauth2-private-key
+  clusterName: test
+  autoAck: true
+  volumeMounts:
+    - mountPath: /mnt/secrets
+      name: sn-platform-oauth2-private-key
+      readOnly: true
```
Comment (tpiperatgod) [inline .ci/tests/integration-oauth2/cases/py-download-function-legacy/manifests.yaml:43]: I think `volumes[0].name` and `volumeMounts[0].name` should be the same

**Comment by freeznet** [comment]
disable the legacy test case and tracking with https://github.com/streamnative/function-mesh/issues/508

**Review by tpiperatgod — APPROVED** [review]

**Review by jiangpengcheng — APPROVED** [review]

**Pull request merged**